js/src/tooltip.js

```diff
@@ -275,6 +276,10 @@ const Tooltip = (() => {
           $(tip).addClass(ClassName.FADE)
         }
 
+        if (this.config.class) {
```

Comment:
```javascript
if (this.config.class.length > 0) {
```
Would be better because here, we will always pass into this `if` because `''` is a valid string or you can change the default value to `null`

Reply:
Isn't your request still valid @Johann-S? If so @mattcheah please take care of it.

Reply:
@mattcheah made the change by changing the default value to `null` so now that's ok 👍